src/plots/cartesian/autorange.js

```diff
@@ -221,14 +221,18 @@ function makePadFn(ax, max) {
     extrappad = adjustPadForInsideLabelsOnAnchorAxis(extrappad, ax, max);
     extrappad = adjustPadForInsideLabelsOnThisAxis(extrappad, ax, max);
 
+    var pad0 = 0;
+    pad0 = adjustPadForInsideLabelsOnAnchorAxis(pad0, ax, max);
+    pad0 = adjustPadForInsideLabelsOnThisAxis(pad0, ax, max);
```

Comment:
That'll work, but it seems like now the code is doing the same thing multiple times. It looks like both of these functions only use the first arg right at the end "return the greater of the newly-calculated value and this original padding." Can we keep the new lines and replace the two `extrappad = adjustPad...` lines above with just:
```js
extrappad = Math.max(extrappad, pad0);
```
after we finish calculating `pad0`?

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/5325/commits/f0709c319b84d813d82acd3d94afeb3d1709f670.